### PR TITLE
Updated the about page and added test

### PR DESCRIPTION
**Pull request by rebeccaTubman**
- Updated the design of the about us page and added a css file for the
style.
- Added a test to check the page is loaded and returns a status code
of 200.
- Added 2 png files for the github and linkedin href links.

new file:
- home/static/home/CSS/about.css
- home/static/home/github.png
- home/static/home/linkedin.png
- home/tests/test_about_page.py

modified:
- home/templates/home/about.html

### CODE REVIEW

**Comment by danitLevi** [comment]
Looking good

File: home/templates/home/about.html [hunk 1]
```diff
@@ -1,34 +1,87 @@
...
+                        assignments.</p>
+                    <p>This web-based application project is part of a course by the Tel Aviv-Yafo Academic College 
+                        under the guidance of RedHat programmers.</p>
+                    <a href="https://github.com/beyond-io/shlifim">Visit our project on Github</a>
+                </div>
+                <br><br>
+                <div class="contact-us">
+                    <h1>Contact Us</h1>
+                    <br>
+                    <p>We are allways growing!</p>
```
Comment (liornoy) [inline home/templates/home/about.html:24]: Typo - always

Comment (rebeccaTubman) [reply home/templates/home/about.html]: Sorry. I corrected the typo.
Thank you:)

**Comment by avivz450** [comment]
LGTM

File: home/static/home/CSS/about.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.main-bg {
+    text-align: center;
+}
```
Comment (liornoy) [inline home/static/home/CSS/about.css:3]: I see you didn't put in "main-bg" class the background color.
what is the background color of the new about page is then?

File: home/tests.py [hunk 3]
```diff
@@ -423,3 +423,10 @@ def test_process_request_with_logged_out_user(self):
         request_mock = Mock(user=self.LoggedOutUserMock())
         pm.process_view(request_mock, None, None, None)
         assert request_mock.profile is None
+
+
+class TestAboutPage:
+    def test_about_page_url(self, client):
+        url = reverse('about')
+        response = client.get(url)
+        assert response.status_code == 200
```
Comment (idokk1) [inline home/tests.py:432]: LGTM ,
just maybe move this class to another file (according to the new arrangement)

File: home/static/home/CSS/about.css (hunk 2 above)
Comment (rebeccaTubman) [reply home/static/home/CSS/about.css:3]: It's an extend of home/base.html so the color is:
```
.main-bg{
	background-color: #d9b6a3;
}
```
Is this color ok or should I change it?

**Review by ifireball — APPROVED** [review]

**Pull request merged**